gulpfile.js

```diff
@@ -32,7 +32,7 @@ const sassOptions = {
   includePaths: ['./node_modules/normalize.css', './node_modules/prismjs/themes']
 };
 
-const scripts = glob.sync('./src/prototypes/**/index.js').map(entryPoint => ({
+const scripts = glob.sync('./src/**/*/index.js').map(entryPoint => ({
   entryPoint: entryPoint,
```

Comment:
I feel like this might cause issues. This glob was intended to find the entry point for a prototype script but now it would also match something like "src/helpers/foo/index.js" which would not be a prototype entry point.

Reply:
OK. My intention is to allow scripts to be made available to templates. I thought that this would just output any JS files with little effect as they would only be loaded if referenced.

Reply:
The various prototype scripts (i.e. "src/prototype/foo/index.js") are able to import any scripts from outside such as "../abc/def.js").